src/Column/ColumnAggregationFunction.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Ublaboo\DataGrid\Column;
+
+class ColumnAggregationFunction
+{
+	/**
+	 * @var string
+	 */
+	protected $column;
+
+	/**
+	 * @var string
+	 */
+	protected $aggregation_type;
+
+	public static $aggregation_type_sum = 'sum';
+
+	public static $aggregation_type_avg = 'avg';
+
+	public static $aggregation_type_min = 'min';
+
+	public static $aggregation_type_max = 'max';
```

Comment:
I have no idea. I ussually use constants for this... But when on foreign project I always take a look how it's done there, saw public static properties on datagrid.php and without thinking further just got them there as well although these are constants. Will change this ASAP